htdocs/core/class/events.class.php

```diff
@@ -103,6 +103,7 @@ class Events // extends CommonObject
 		array('id'=>'USERGROUP_CREATE', 'test'=>1),
 		array('id'=>'USERGROUP_MODIFY', 'test'=>1),
 		array('id'=>'USERGROUP_DELETE', 'test'=>1),
+		array('id'=>'USER_RIBMODIFY', 'test'=>1),
```

Comment:
RIB is a french term, prefer to use english term like IBAN

Reply:
Done, but in a great part of code the term RIB is used... If you think is a good  idea I can explore the whole source code to transform RIB in IBAN